From mgreer at mvista.com  Fri Apr 24 21:18:45 2009
From: mgreer at mvista.com (Mark A. Greer)
Date: Fri Apr 24 21:14:20 2009
Subject: [PATCH 1/3 v4] ARM: DaVinci - Remove unused compare reg defines
Message-ID: <20090425021845.GA14176@mag.az.mvista.com>

From: Mark A. Greer <mgreer@mvista.com>

The offset for the compare register used by the davinci
time code are passed in via the cmp_off member of the
'davinci_timer_instance' structure.  Therefore, the
offset defined in time.c are not necessary.

Signed-off-by: Mark A. Greer <mgreer@mvista.com>
---
This patch is same as previous version but I didn't want
it to get lost so I resubmitted.

 arch/arm/mach-davinci/time.c |   10 ----------
 1 files changed, 0 insertions(+), 10 deletions(-)

diff --git a/arch/arm/mach-davinci/time.c b/arch/arm/mach-davinci/time.c
index e72da73..6297302 100644
--- a/arch/arm/mach-davinci/time.c
+++ b/arch/arm/mach-davinci/time.c
@@ -54,16 +54,6 @@ enum {
 #define TGCR			0x24
 #define WDTCR			0x28
 
-/* Offsets of the 8 compare registers */
-#define	CMP12_0			0x60
-#define	CMP12_1			0x64
-#define	CMP12_2			0x68
-#define	CMP12_3			0x6c
-#define	CMP12_4			0x70
-#define	CMP12_5			0x74
-#define	CMP12_6			0x78
-#define	CMP12_7			0x7c
-
 /* Timer register bitfields */
 #define TCR_ENAMODE_DISABLE          0x0
 #define TCR_ENAMODE_ONESHOT          0x1
-- 
1.6.0.3
